nautobot/core/templates/about.html

```diff
@@ -0,0 +1,67 @@
+{% extends 'base.html' %}
+{% load helpers %}
+{% load registry %}
+
+{% block header %}
+    {{ block.super }}
+    {% if new_release %}
+        {# new_release is set only if the current user is a superuser or staff member #}
+        <div class="alert alert-info text-center" role="alert">
+            <i class="mdi mdi-information-outline"></i>
+            A new release is available: <a href="{{ new_release.url }}">Nautobot v{{ new_release.version }}</a> |
+            <a href="https://docs.nautobot.com/projects/core/en/v{{ new_release.version }}/installation/upgrading/">Upgrade instructions</a>
+        </div>
+    {% endif %}
+{% endblock %}
+
+
+{% block content %}
+    <h1>About Nautobot</h1>
+    <div class="row">
+        <div class="col-md-6">
+            <div class="panel panel-default">
+                <div class="panel-heading">
+                    <strong>About</strong>
```

Comment:
Should the `About` panel display the active Nautobot version prominently somewhere?

Reply:
I went back and forth on that, as you can see from having the upgrade check included. I'll add it back and we can see.